proto/store/setting.proto

```diff
@@ -79,3 +79,21 @@ message SMTPMailDeliverySetting {
   // The sender email address.
   string from = 10;
 }
+
+message AppIMSetting {
```

Comment:
messages should be PascalCase, AppImSetting in thie case

Reply:
how do you think of
https://github.com/bytebase/bytebase/blob/main/proto/store/user.proto#L8
https://github.com/bytebase/bytebase/blob/main/proto/store/sheet.proto#L8
looks like in our code style we will use FULL upper case for abbreviation

Reply:
That is unfortunate...
I think we should stick to PascalCase and fix those.

Reply:
By the way, it's strange that buf lint didn't spot them.
We have the rules activated. https://buf.build/docs/lint/rules/#enum_pascal_case